src/libmongoc/src/mongoc/mongoc-client-side-encryption.c

```diff
@@ -1514,8 +1565,12 @@ mongoc_client_encryption_new (mongoc_client_encryption_opts_t *opts,
    mongoc_collection_set_write_concern (client_encryption->keyvault_coll, wc);
 
    client_encryption->kms_providers = bson_copy (opts->kms_providers);
-   client_encryption->crypt = _mongoc_crypt_new (
-      opts->kms_providers, NULL /* schema_map */, opts->tls_opts, error);
+   client_encryption->crypt = _mongoc_crypt_new (opts->kms_providers,
+                                                 NULL /* schema_map */,
+                                                 opts->tls_opts,
+                                                 NULL /* No csfle path */,
+                                                 false /* csfle not requried */,
+                                                 error);
```

Comment:
This should also pass information through to disable loading `csfle`. It is not needed for a `mongoc_client_encryption_t`. This is new behavior from DRIVERS-2287.